cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,47 @@
 /// <reference types='cypress' />
 
+const user = {
```

Comment:
you know how to generate random user data - create generateUser function and reuse it in the test